src/lightning_graphcore/utils.py

```diff
@@ -0,0 +1,61 @@
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Union
+
+import lightning.pytorch as pl
```

Comment:
this shall also acceplt PL 
```suggestion
if package_available("lightning"):
    import lightning.pytorch as pl
elif package_available("pytorch_lightning"):
    import pytorch_lightning as pl
```